### DISCUSSION TITLE
One attachment per declination

### DISCUSSION THREAD

**Discussion by LSHRVM** [PrestaShop/PrestaShop#40180] [Ideas]
Hello,

I think there is a missing feature with declination.

As a declination is not only a color, or just optionals features on one product but could be a range of product from a same series, my interest would be to add one attachment per declination.

As this feature is not existing, I request here the possibility to integrate this feature on a next release or on an addon.

Thank you for your attention.

Best regards,
Ludovic

[a comment by an automated account is not shown]

**Comment by florine2623** [comment]
Hi @LSHRVM ,
Thanks for the suggestion ! Please stay tuned.

**Comment by florine2623** [comment]
Hi @PrestaShop/product-team ,
What do you think of having one attachment per combination ?
Thanks !

**Comment by LSHRVM** [comment]
Hello,

Thank you for your suggestion but the problem is the same with combination, we can't attach one PDF per combination. I saw that there is an addon for that but all my datas are already recorded as declination (6600 SKU...). Also, is each product ref will be referenced for SEO or only the parent product ?

Best regards,
Ludovic

**Comment by SimonGrn** [comment]
Hello @LSHRVM 
if you're talking about a third-party module, we cannot help you with that here, you need to contact the module developer. Please remember that Github is not a place for support but only for issues with the core of PrestaShop.

We need an answer for our Product Team to decide if it should be a Core feature or not.

Please stay tuned.

Thank you.

**Comment by LSHRVM** [comment]
Dear Simon,

I don't request for a third-party module. I think that this feature need to be in the Core.

Best regards,
Ludovic

**Comment by SimonGrn** [comment]
I was talking about this part in your previous comment : 
>I saw that there is an addon for that

We will see what the Product Team decides. Please remember that 1.7.7.x is feature-frozen so this feature would be a minima in the 1.7.8.x branch which is scheduled for late 2021/start 2022.
